.github/workflows/backwards_compatibility_marqo_execution.yml

```diff
@@ -53,11 +57,16 @@ on:
           - rollback
         required: true
         type: choice
+      path_to_test:
+        description: 'Path sent to pytest for running test mode'
```

Comment:
Can you add an example in the description to show what the path should look like?